src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/pact/provider/ManageRecallsUIPactTest.kt

```diff
@@ -121,11 +130,20 @@ class ManagerRecallsUiAuthorizedPactTest : ManagerRecallsUiPactTestBase() {
     "a user and a fully populated recall without documents exists"
   )
   fun `a user and a fully populated recall without documents exists`() {
-    val recall = fullyPopulatedRecall(::RecallId.zeroes(), userIdOnes).copy(
-      documents = emptySet(),
-      missingDocumentsRecords = emptySet(),
-      assignee = userIdOnes.value
-    )
+    val recall = fullyPopulatedRecall(::RecallId.zeroes(), userIdOnes).let {
+      it.copy(
+        currentPrison = PrisonId("BMI"),
+        lastReleasePrison = PrisonId("CFI"),
+        sentencingInfo = it.sentencingInfo?.copy(sentencingCourt = CourtId("BANBCT")),
+        localPoliceForceId = PoliceForceId("avon-and-somerset"),
+        licenceNameCategory = NameFormatCategory.FIRST_LAST,
```

Comment:
In order to generate revocation order, these values need to be set and in the first 4 align with values in the Mocks